src/Device/Driver/LX_EOS/LXEosDevice.hpp

```diff
@@ -16,6 +16,85 @@
 
 using std::string_view_literals::operator""sv;
 
+struct EosDeclarationStruct
+{
+  const uint8_t syn = 0x02;
+  const uint8_t cmd = 0xCA;
+  const uint8_t flag = 0;   // Not used
+  const uint16_t oo_id = 0; // Not used
+  char pilot[19];           // "Name Surname"
+  char glider[12];          // Polar name
+  char reg_num[8];          // Glider registration number
+  char cmp_num[4];          // Competition id
+
+  // 0=STANDARD, 1=15-METER, 2=OPEN, 3=18-METER,
+  // 4=WORLD, 5=DOUBLE, 6=MOTOR_GL
+  uint8_t byClass;
+  char observer[10];              // Not used
+  const uint8_t gpsdatum = 0;     // Not used
+  const uint8_t fix_accuracy = 0; // Not used
+  char gps[60];                   // Not used
+
+  /* auto defined */
+  const uint8_t flag2 = 0;      // Not used
+  const int32_t input_time = 0; // Not used
+  const uint8_t di = 0;         // Not used
+  const uint8_t mi = 0;         // Not used
+  const uint8_t yi = 0;         // Not used
+  /* user defined */
+  const uint8_t fd = 0;     // Not used
+  const uint8_t fm = 0;     // Not used
+  const uint8_t fy = 0;     // Not used
+  const int16_t taskid = 0; // Not used
+
+  // Number of TP without Takeoff, Start, Finish and Landing.
+  char num_of_tp;
+
+  // 1=Turnpoint (also Start and Finish), 2=Landing, 3=Takeoff
+  uint8_t prg[12];
+
+  // TP Longitude in degrees multiplied by 60000.0f
+  int32_t lon[12];
+
+  //< TP Latitude in degrees multiplied by 60000.0f
+  int32_t lat[12];
+
+  char name[12][9]; //< TP Name
+
+  uint8_t crc;
+} gcc_packed;
+
+struct EosObsZoneStruct
+{
+  const uint8_t syn = 0x02;
+  const uint8_t cmd = 0xF4;
+
+  // TP number 1=Start, 2 = TP1, 3=TP2, 4=Finish
+  uint8_t uiTpNr;
+
+  // direction 0= Symmetric, 1=Fixed, 2=Next, 3=Previous
+  uint8_t uiDirection;
+
+  uint8_t bAutoNext = 1; // Is this auto next TP or AAT TP
+  uint8_t bIsLine = 0;   // Is this line flag
+  float fA1;             // Angle A1 in radians
```

Comment:
Byte order? What bit format is being used? Is this bit format guaranteed on all CPU architectures?